languages/python/templates/api_doc.mustache

```diff
@@ -1,3 +1,5 @@
+{{! This file is picked from the v5.1.0 release of openapi-generator(https://github.com/OpenAPITools/openapi-generator/blob/v5.1.0/modules/openapi-generator/src/main/resources/python/api_doc.mustache) }}
```

Comment:
Any reason why changes are not reflected in this template file?

Reply:
Couldn't get it working properly unfortunately. The markdown generated can't even display properly and would need more time to figure out why.